### PR TITLE
Add script to convert tutorials to Python files

### LINKED ISSUE #20: Link python files to tutorials
For each tutorial on the website, provide a python file for the user to download. We'd like to automate this process to avoid needing to keep track of updates as content on the website changes. [Mkdocs gallery](https://smarie.github.io/mkdocs-gallery/) may be a suitable solution. @melissawm

### PR DESCRIPTION
**Pull request by melissawm**
Fixes #20

For some reason, the glob syntax I was using was skipping a few files, so I decided to go with a more explicit script.

### CODE REVIEW

**Comment by melissawm** [comment]
I've tested this on my fork, it should be good to go now. Cheers!

File: .github/workflows/convert.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+echo "Converting all .md tutorials to .py files..."
+cd ./docs/tutorials;
```
Comment (aliddell) [inline .github/workflows/convert.sh:3]:
```suggestion
TUTS_DIR="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../../docs/tutorials"
pushd ${TUTS_DIR}
```

I made a suggestion on Slack that was wrong, but this one [appears to be reliable](https://stackoverflow.com/a/4774063).

File: .github/workflows/convert.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+for file in *.md; do
+    if [ "$file" != "index.md" ]; then
+        jupytext --to py $file
+    fi
+done
+cd -
```
Comment (aliddell) [inline .github/workflows/convert.sh:9]:
```suggestion
popd
```
if you go with  the other suggestion you should match the `pushd` with a `popd`

**Review by aliddell — APPROVED** [review]
I'm ok to push this one forward if you think it's likely this script will only ever be called in CI. Otherwise, I added a little code to ensure that no matter where it's called you can get the absolute path to the tutorials directory.

**Comment by melissawm** [comment]
Thanks @aliddell - that's a good idea 😄

**Pull request merged**